documentapi/src/main/java/com/yahoo/documentapi/Result.java

```diff
@@ -85,5 +111,16 @@ public enum ResultType {
         @Deprecated(since = "7", forRemoval = true) // TODO: Remove on Vespa 8 — this is a Response outcome, not a Result outcome.
         CONDITION_NOT_MET_ERROR
     }
+    public static Error toError(ResultType result) {
+        switch (result) {
+            case TRANSIENT_ERROR:
+                return new Error(ErrorCode.TRANSIENT_ERROR, ResultType.TRANSIENT_ERROR.name());
+            case CONDITION_NOT_MET_ERROR:
```

Comment:
Condition not met isn't transient?

Reply:
No you will need to change your condition for it to have a bigger chance at success, and anyway it should not be here. See comment in Result.ResultType.